### DISCUSSION TITLE
ios labels are cut off when wrapped

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3686] [Mobile] [answered]
iphone 7 ios 14.3, labels are cut off when wrapped to the beginning of the line

| 1 | 2 |
| - | - |
| ![IMG_0203](https://user-images.githubusercontent.com/39104740/113487605-c6cf3780-94c1-11eb-94a3-6b55ff45701c.PNG) | ![IMG_0196](https://user-images.githubusercontent.com/39104740/113487610-cc2c8200-94c1-11eb-97b4-8ef93b9269e0.PNG) |

**Comment by brianlovin** [comment, marked as answer]
Bug report filed, thanks so much!
